db/migrate/20170530173726_add_filepath_to_submissions.rb

```diff
@@ -0,0 +1,5 @@
+class AddFilepathToSubmissions < ActiveRecord::Migration[5.1]
+  def change
+    add_column :submissions, :filepath, :string
+  end
+end
```

Comment:
Don't do this. Since we are rebuilding our db every time, just directly change the original create table migration.